src/ui/Misc/Slider.ts

```diff
@@ -809,46 +832,42 @@ class SliderGraph {
   }
 
   private setTooltip(d: ISliderGraphData, height: number) {
-    const caption = $$('span', {
-      className: 'coveo-caption'
-    });
-    caption.text(this.slider.getCaptionFromValue([d.start, d.end]));
+    $$(this.tooltipCaption).text(this.slider.getCaptionFromValue([d.start, d.end]));
+    $$(this.tooltipCount).text(d.y.toString());
+    $$(this.tooltip).show();
 
-    const count = $$('span', {
-      className: 'coveo-count'
-    });
-    count.text(d.y.toString());
-    $$(this.tooltip).empty();
-    this.tooltip.appendChild(caption.el);
-    this.tooltip.appendChild(count.el);
+    const leftPositionForCurrentBand = this.x(d.start) - 0.2 * this.slider.options.graph.steps;
```

Comment:
I wonder if these formulas be more expressive? Not sure what `this.x(d.start)` or `0.2` mean. Similarly, why do we subtract `5` from `halfOfBandwidth`?